public/services/shared-link.ts

```diff
@@ -107,8 +107,12 @@ export function setClipboardAndTarget(
   const referenceNode = document.getElementsByTagName('span').item(0);
 
   range.selectNode(referenceNode);
-  shareButton.removeAllRanges();
-  shareButton.addRange(range);
+  try {
+    shareButton.removeAllRanges();
+    shareButton.addRange(range);
+  } catch (e) {
+    /* ignore */
```

Comment:
In my testing, I get `TypeError: shareButton.removeAllRanges is not a function` which means that this function isn't being executed as expected on Chrome. It may work on other browsers, but it does not work on chrome in my testing.

Reply:
@joshuarrrr @kavilla Seems like there should be a polyfill for this... but this is outside of my wheelhouse, do you guys have an idea?

Reply:
@peternied I pushed another commit. This doesn't need to be polyfilled, it needs to be called on the correct object type. 

According to MDN, this should be called on a Selection: https://developer.mozilla.org/en-US/docs/Web/API/Selection/removeAllRanges